### ISSUE TITLE
Broken link in Bin2Dec

### ISSUE THREAD

**Issue by ahujaesh** [florinpop17/app-ideas#818]
**Describe the bug**

**To Reproduce**
1. Go to the readme.md file
2. Click on the link on line 46 - Bin2Dec ( [here](https://github.com/florinpop17/app-ideas/blob/master/README.md?plain=1#L46) )
3. Scroll down to "Binary to Decimal converter with plain html, js and css"
4. Click the link
5. See error - GitHub pages site not found

**Expected behavior**
I expected to be taken to an example, but the [link](https://grfreire.github.io/Bin2Dec/) is broken (There is no website there).

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by titiksha6393** [comment]
Yes even the link to [Matrix-like Binary to Decimal converter using Angular](https://github.com/ZangiefWins/MatrixBin2Dec)
[Live preview on heroku](https://matrix-bin2dec.herokuapp.com/) is not working there is nothing there
